pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +46,24 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Add Current Profile Context
+func AddContextForProfile(profile string, ip string) error {
+	cmdName := instanceState.Config.OcPath
+	ip = strings.Replace(ip, ".", "-", -1)
+	cmdArgs := []string{"config", "set-context", profile,
+		fmt.Sprintf("--cluster=%s:%d", ip, constants.APIServerPort),
+		fmt.Sprintf("--user=developer/%s:%d", ip, constants.APIServerPort),
+		"--namespace=myproject"}
```

Comment:
Would it make sense to run first:

    $ oc config set-cluster minishift --server=https://192.168.99.105:8443